pkg/kubelet/eviction/helpers.go

```diff
@@ -1263,13 +1267,13 @@ func evictionMessage(resourceToReclaim v1.ResourceName, pod *v1.Pod, stats stats
 				}
 				if usage != nil && usage.Cmp(requests) > 0 {
 					message += fmt.Sprintf(containerMessageFmt, container.Name, usage.String(), requests.String(), resourceToReclaim)
-					containers = append(containers, container.Name)
+					exceededContainers = append(exceededContainers, container.Name)
 					containerUsage = append(containerUsage, usage.String())
 				}
 			}
```

Comment:
why there is no break out of the loop when the name match found?

Reply:
I was wondering about that at first too, but I figured it’s probably because multiple containers might be exceeding their resource requests.

Reply:
Yes, but it will be covered by the outer loop. Inner loop is a lookup of a container by it's name and it should be broken out of when the container with the matching name was found

Reply:
Thanks for the explanation! I get it now. Since both the original implementer and I didn't get it right away, I added a comment, even though it might be a bit redundant.